src/style/validate_style.js

```diff
@@ -6,10 +6,12 @@ import type {Evented} from '../util/evented.js';
 
 type ValidationError = {
     message: string,
-    line: number,
+    line?: number,
     identifier?: string
 };
 
+export type ValidationErrors = $ReadOnlyArray<ValidationError>;
```

Comment:
We may also use `$Shape` instead of making the `ValidationError.line` optional.

```ts
export type ValidationErrors = $ReadOnlyArray<$Shape<ValidationError>>;
```

Also it seems that this type may be replaced with the [style-spec ValidationError](https://github.com/mapbox/mapbox-gl-js/blob/3483ba7f4268422dec4728a37cb432dfe68888fb/src/style-spec/error/validation_error.js#L5)